components/scream/src/physics/p3/atmosphere_microphysics.cpp

```diff
@@ -175,7 +175,20 @@ void P3Microphysics::init_buffers(const ATMBufferManager &buffer_manager)
 void P3Microphysics::initialize_impl (const RunType /* run_type */)
 {
   // Set property checks for fields in this process
-  add_invariant_check<FieldWithinIntervalCheck>(get_field_out("T_mid"),140, 500);
+  add_invariant_check<FieldWithinIntervalCheck>(get_field_out("T_mid"),140.0, 500.0);
```

Comment:
Haven't been able to verified if these checks are functioning properly. Purposely resetting the T_mid bound checks between 140 and 270 didn't trigger any warning or fatal error in a multi-step run.